src/app/modules/spcp/spcp.types.ts

```diff
@@ -43,3 +45,8 @@ export interface ParsedSpcpParams {
   rememberMe: boolean
   cookieDuration: number
 }
+
+export interface ISPCPForm extends IFormSchema {
```

Comment:
nit: rename to `ISpcpForm`, we tend to not use UPPERCASE for abbreviations/initialisms